etc/profile-m-z/minecraft-launcher.profile

```diff
@@ -54,7 +56,9 @@ private-etc @tls-ca,@x11,host.conf,java*,mime.types,services,timezone
 private-opt minecraft-launcher
 private-tmp
 
-dbus-user none
+dbus-user.talk org.freedesktop.secrets
```

Comment:
Without `dbus-user filter` the 3 `dbus-user.talk` rules don't get applied. So in fact the complete session bus is now allowed in the sandbox. Can you confirm that adding `dbus-user filter` above this line does allow keyring access?

Reply:
I'll definitely check tomorrow

Reply:
It works.